components/contribution-flow/ContributionFlowStepContainer.js

```diff
@@ -187,21 +192,23 @@ class ContributionFlowStepContainer extends React.Component {
   };
 
   render() {
-    const { LoggedInUser, step } = this.props;
+    const { LoggedInUser, step, isCrypto } = this.props;
 
     return (
       <StyledCard p={[16, 32]} mx={[16, 'none']} borderRadius={15}>
         <Flex flexDirection="column" alignItems="center">
-          <Flex width="100%" mb={3}>
-            <Flex alignItems="center">
-              <H4 fontSize={['20px', '24px']} fontWeight={500} py={2}>
-                {this.renderHeader(step.name, LoggedInUser)}
-              </H4>
-            </Flex>
-            <Flex flexGrow={1} alignItems="center" justifyContent="center">
-              <StyledHr width="100%" ml={3} borderColor="black.300" />
+          {!(step.name === 'payment' && isCrypto) && (
```

Comment:
Why is this necessary?

Reply:
@znarf : Note that according to the design we don't show a section header for the checkout/payment screen. We just show the QR code. Now I did the QR code addition in a separate PR, but wanted to make the other parts of the contribution flow as close as possible to the design.

https://www.figma.com/file/ZQBMWhnGGtRWeIZknFW1eP/%5BOC-Design%5D-Production-Ready-%E2%9C%85?node-id=13426%3A207907